eth/beacon/deposit_helpers.py

```diff
@@ -0,0 +1,160 @@
+from typing import (
+    Sequence,
+    Tuple,
+)
+
+from eth_typing import (
+    Hash32,
+)
+from eth_utils import (
+    ValidationError,
+)
+
+from eth.utils import bls
+
+from eth.beacon.constants import (
+    EMPTY_SIGNATURE,
+)
+from eth.beacon.enums import (
+    SignatureDomain,
+)
+from eth.beacon.exceptions import (
+    MinEmptyValidatorIndexNotFound,
+)
+from eth.beacon.types.deposit_input import DepositInput
+from eth.beacon.types.states import BeaconState
+from eth.beacon.types.validator_records import ValidatorRecord
+from eth.beacon.helpers import (
+    get_domain,
+)
+
+
+def get_min_empty_validator_index(validators: Sequence[ValidatorRecord],
+                                  current_slot: int,
+                                  zero_balance_validator_ttl: int) -> int:
+    for index, validator in enumerate(validators):
+        if (
```

Comment:
small formatting nitpick.  I'd capture the contidional in a variable and then use it in a single line `if`

```python
is_empty = (validator.balance == 0 and ...)
if is_empty:
    return index
```